doc/cascadia/profiles.schema.json

```diff
@@ -873,11 +873,29 @@
             "string",
             "null"
           ]
+        },
+        "commands":{
```

Comment:
we also need to add `iterateOn` which would be an enum that only accepts `profiles` or `schemes`.

Doc: https://docs.microsoft.com/en-us/windows/terminal/command-palette#iterable-commands